changelog.d/templates/new_fragment.md.j2

```diff
@@ -4,7 +4,8 @@
 ### {{ cat }}
 
 - Describe change, possibly reference closed/related issue/PR.
-  Fixes https://github.com/datalad/datalad-next/issues/XXX via
-  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+  Fixes [#XXX](https://github.com/datalad/datalad-next/issues/XXX) via
+  [PR #YYY](https://github.com/datalad/datalad-next/pull/YYY)
+  (by [@GITHUBHANDLE](https://github.com/GITHUBHANDLE))
```

Comment:
we might want to dewarp it into one line since otherwise `scriv github` release notes are "very suboptimally formatted". Discussion/investigation is ongoing in https://github.com/datalad/release-action/issues/8

Reply:
crap, just merged and saw that it is still `datalad-next`, I will fix and push directly